client/src/components/Sidebar/Sidebar.js

```diff
@@ -103,6 +104,9 @@ const Sidebar = () => {
           </NavLink>
         </NavItem>
       )}
+      <NavItem styleName="feedback-nav-item">
+        <FeedbackButton />
+      </NavItem>
```

Comment:
For some reason this does not appear in the sidebar while in the `/allocations` view

Reply:
@jmartinez-tacc , note: the allocations route changed in master (and now in your branch) in the last hour.

Reply:
Can I see an /allocations screenshot?

Reply:
Thank you @nathanfranklin!
@rstijerina, this button shows at the bottom of the screen, I just wanna make sure this is not happening because `/allocations` content is too big that scrolling down is needed. Could you confirm?

Reply:
You're correct! Thanks for pointing that out